source/agora/test/Base.d

```diff
@@ -1932,23 +1958,6 @@ public struct TestConf
     /// The minimum (transaction size adjusted) fee.
     /// Transaction size adjusted fee = tx fee / tx size in bytes.
     public Amount min_fee = Amount(0);
-
-    /// The maximum number of transactions relayed in every batch.
-    /// Value 0 means no limit.
-    uint relay_tx_max_num = 0;
-
-    /// Transaction relay batch is triggered in every `relay_tx_interval`.
-    /// Value 0 means, the transaction will be relayed immediately.
-    Duration relay_tx_interval = 0.seconds;
-
-    /// The minimum amount of fee a transaction has to have to be relayed.
-    /// The fee is adjusted by the transaction size:
-    /// adjusted fee = fee / transaction size in bytes.
-    Amount relay_tx_min_fee = 0;
-
-    /// Transaction put into the relay queue will expire, and will be removed
-    /// after `relay_tx_cache_exp`.
-    Duration relay_tx_cache_exp = 60.minutes;
```

Comment:
Isn't it contained in the `node` of `TestConf`?

Reply:
You mean, the `60.minutes` value ? Well, I looked at the configuration, and the reason why we have 60 minutes is to ensure that we don't evict anything, essentially. But the default already does that. Ping @ferencdg to check my sanity.